sources/packages/web/src/views/student/application-offering-change-request/ApplicationOfferingChangeFormView.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+  <full-page-container layout-template="centered-card-tab">
+    <template #header>
+      <header-navigator
+        title="Financial Aid Application"
+        sub-title="View Request"
+        :routeLocation="goBackRouteParams"
+      >
+        <template
+          #buttons
+          v-if="
+            applicationOfferingChangeRequestStatus ===
+            ApplicationOfferingChangeRequestStatus.InProgressWithStudent
+          "
+        >
+          <v-btn
+            v-if="showActionButtons"
+            color="primary"
+            variant="outlined"
+            @click="declineApplicationOfferingChangeRequest"
+            >Decline change</v-btn
+          >
+          <v-btn
+            v-if="showActionButtons"
+            color="primary"
+            class="ml-2"
+            @click="approveApplicationOfferingChangeRequest"
+            >Allow change</v-btn
+          >
+        </template>
+      </header-navigator>
+    </template>
+    <template #tab-header>
+      <v-tabs stacked color="primary"
+        ><v-tab
+          v-for="item in items"
+          :key="item.label"
+          :to="item.command()"
+          :ripple="false"
+        >
+          <span class="mx-2 label-bold"> {{ item.label }} </span>
+        </v-tab>
+      </v-tabs>
+    </template>
+    <router-view />
+    <approve-application-offering-change-request-modal
```

Comment:
I am having the below error in my local, pls take a look, 
![image](https://github.com/bcgov/SIMS/assets/77353155/d13feb36-280a-4449-9f6e-7dc945785e52)

Reply:
Are we ignoring the above errors? I see similar error pre-existing for `ApproveProgramModal` and `DeclineProgramModal` in the `ProgramDetails.vue`. Should I correct in other places too as a part of this PR? @dheepak-aot @andrepestana-aot @guru-aot suggestions?

Reply:
IMO. If it takes too long to investigate this issue, I would say it could be done as seperate effort as we have all the existing models having this issue and have all of them fixed at once.

Reply:
Agree with @dheepak-aot, If its taking time to fix the issue, pls create a ticket for it

Reply:
https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/2224